Gems/Terrain/Code/Tests/TerrainPhysicsColliderTests.cpp

```diff
@@ -141,6 +138,9 @@ TEST_F(TerrainPhysicsColliderComponentTest, TerrainPhysicsColliderReturnsAligned
     const float boundsMin = 0.0f;
     const float boundsMax = 1024.0f;
 
+    // "max - min" gives us the number of grid squares, "max - min + 1" gives us the number of grid vertices including the final endcap.
+    const int32_t expectedGridSize = aznumeric_cast<int32_t>(boundsMax - boundsMin) + 1;
```

Comment:
nit / repeated: it looks like this is only used in the EXPECT_EQ at the end of the function. It'd be better to place this right before where it's used.

Reply:
Not sure I agree that it's _better_, I liked having it initialized next to boundsMin / boundsMax since they were all related, but I don't feel strongly about it so I moved them down.